draft-ietf-tls-tls13.md

```diff
@@ -4037,6 +4046,28 @@ lacking a "server_name" extension by terminating the connection with a
 Security issues are discussed throughout this memo, especially in
 {{implementation-notes}}, {{backward-compatibility}}, and {{security-analysis}}.
 
+## Re-Using Ephemeral Shares
+
+This document refers to client and server key shares as ephemeral. This literally
+means that they are used for a short time. Keeping the private key secret forever
+is necessary for the property of Perfect Forward Secrecy. The simplest and safest 
+way to achieve this is to generate a new Diffie-Hellman share for each TLS 
+handshake, and to destroy keys immediately after they are used. This has some 
+cost in processing power. 
+
```

Comment:
@tvdmerwe: I believe the properties wrt the RMS are as follows:

- Knowledge of the RMS established in connection N is not sufficient to decrypt N
- If you establish a new connection N+1 with the RMS as the PSK and do not do DH, then knowledge of the RMS is sufficient to decrypt N+1. However, N+1 generates its own RMS and that is not sufficient to decryption N+1 (though of course you can use it for N+2).
- If you operate a session cache type system, then the server can delete the RMS once it has been used to establish the session, giving you PFS even w/o DH
- If you operate a ticket-based system, you don't get PFS until the ticket key has been destroyed.